tests/e2e/framework/istioctl.go

```diff
@@ -183,7 +183,7 @@ func (i *Istioctl) DeleteRule(rule string) error {
 }
 
 // PortEPs is a map from port number to a list of endpoints
-type PortEPs map[string][]string
+type PortEPs map[string]map[string]int
```

Comment:
nit: you don't really use the integer. It could be just `map[string]map[string]struct{}`

Reply:
I do, at line 328 it increments the integer.

Reply:
right, but nothing reads the value ... unless I'm missing something.